config/crd/bases/config.ratify.deislabs.io_keymanagementproviders.yaml

```diff
@@ -50,6 +50,9 @@ spec:
           spec:
             description: KeyManagementProviderSpec defines the desired state of KeyManagementProvider
             properties:
+              interval:
```

Comment:
Suggestion to add some samples of refresh interval configured AKV in the samples dir thanks! , https://github.com/ratify-project/ratify/tree/dev/config/samples